src/room/LobbyView.tsx

```diff
@@ -129,16 +129,16 @@ export const LobbyView: FC<Props> = ({
         <div className={inCallStyles.footer}>
           {recentsButtonInFooter && recentsButton}
           <div className={inCallStyles.buttons}>
-            <VideoButton
-              muted={!muteStates.video.enabled}
-              onPress={onVideoPress}
-              disabled={muteStates.video.setEnabled === null}
-            />
             <MicButton
               muted={!muteStates.audio.enabled}
               onPress={onAudioPress}
               disabled={muteStates.audio.setEnabled === null}
             />
+            <VideoButton
+              muted={!muteStates.video.enabled}
+              onPress={onVideoPress}
+              disabled={muteStates.video.setEnabled === null}
+            />
```

Comment:
why do we not neet the `key=1 | 2` here?

Reply:
React only requires keys when you use an array of ReactElements as a child, because then it can't statically know which part of the code creates which child. But in this case, since we don't need to dynamically create a buttons array, React knows that the order of these elements is stable.

Reply:
The question was mostly on what makes the incallview different.
If it is in a different container incall vs lobby there is different behaviour for the buttons in the two views. What makes it necassary to have different behaviour in those two view?

Reply:
The question was mostly on what makes the incallview different.
If it is in a different container incall vs lobby there is different behaviour for the buttons in the two views. What makes it necassary to have different behaviour in those two view?